src/uu/lsmem/src/lsmem.rs

```diff
@@ -516,16 +581,31 @@ where
 #[uucore::main]
 pub fn uumain(args: impl uucore::Args) -> UResult<()> {
     let matches: clap::ArgMatches = uu_app().try_get_matches_from(args)?;
-    let opt_bytes = matches.get_flag(options::BYTES);
 
     let mut lsmem = Lsmem::new();
     let mut opts = Options::new();
-    opts.bytes = opt_bytes;
+    opts.bytes = matches.get_flag(options::BYTES);
+    opts.noheadings = matches.get_flag(options::NOHEADINGS);
+    opts.json = matches.get_flag(options::JSON);
+    opts.export = matches.get_flag(options::PAIRS);
```

Comment:
Why do you name this option "export" and not "pairs"?

Reply:
In the original explantation, it is named as export 
https://github.com/util-linux/util-linux/blob/master/sys-utils/lsmem.c#L617